modules/services/getmail.nix

```
@@ -44,7 +44,7 @@ in {
 
     systemd.user.services.getmail = {
       Unit = { Description = "getmail email fetcher"; };
-      Service = { ExecStart = "${pkgs.getmail}/bin/getmail ${configFiles}"; };
+      Service = { ExecStart = "${pkgs.getmail6}/bin/getmail ${configFiles}"; };
```

```suggestion
      Service = { ExecStart = "${pkgs.getmail6 or pkgs.getmail}/bin/getmail ${configFiles}"; };
```

If the package isn't available in stable nixpkgs then I recommend the patch above.

---

this is a bit confusing and makes troubleshooting harder I think. I would rather merge as is. HM master is supposed to target unstable anyway

---

Yeah I was wondering whether we should not better make it explicit (`assert`?) that `getmail` is deprecated and the user has to upgrade. Fine with me either way.

---

I think the nix error is clear enough. Let's not dwell on this.